aries_cloudagent/admin/server.py

```diff
@@ -131,7 +132,24 @@ async def ready_middleware(request: web.BaseRequest, handler: Coroutine):
         "/status/live",
         "/status/ready",
     ) or request.app._state.get("ready"):
-        return await handler(request)
+        try:
+            return await handler(request)
+        except LedgerConfigError:
+            # fatal, signal server shutdown
+            LOGGER.error(f"Shutdown with LedgerConfigError")
+            request.app._state["ready"] = False
+            request.app._state["alive"] = False
+            raise
+        except LedgerTransactionError:
+            # fatal, signal server shutdown
+            LOGGER.error(f"Shutdown with LedgerTransactionError")
+            request.app._state["ready"] = False
+            request.app._state["alive"] = False
+            raise
+        except Exception as e:
+            # some other error?
+            LOGGER.error("Handler error with exception: " + str(e))
```

Comment:
Log statements should use this format: `LOGGER.error("msg: %s", arg)` (also the f-strings above are not necessary)